fail2ban/server/database.py

```diff
@@ -201,6 +216,11 @@ def __init__(self, filename, purgeAge=24*60*60):
 						": updated from '%i' to '%i'",
 						Fail2BanDb.__version__, version, newversion)
 					raise RuntimeError('Failed to fully update')
+
+			# pypy: set journal mode after possible upgrade db:
+			if pypy:
+				cur.execute("PRAGMA journal_mode = MEMORY")
+
```

Comment:
Thx, you are right (it was done only by upgrade, what is very rare), moved in finally scope now